letsencrypt/auth_handler.py

```diff
@@ -50,6 +50,8 @@ def __init__(self, dv_auth, cont_auth, acme, account):
         self.account = account
         self.authzr = dict()
 
+        self.config = config
```

Comment:
nit: this is randomly placed and uses random whitespace around - please move above, under `self.acme = acme`